### PR TITLE
README.rst: Update broken links

### PR DESCRIPTION
**Pull request by ashwani99**
Fixes https://github.com/coala/coala-bears/issues/1159

### CODE REVIEW

**Comment by sils** [comment]
ack https://github.com/coala/coala-bears/pull/1174/commits/0ec39bd9b1bcc04e11c9550e2b02cc601af3065d

**Comment by Mixih** [comment]
@rultor merge

**Comment by Mixih** [comment]
please rebase

**Comment by Asnelchristian** [comment]
ack https://github.com/coala/coala-bears/pull/1174/commits/869de36fc25171210158a65f93ab5493912271c9

File: bears/general/InvalidLinkBear.py [hunk 1]
```diff
@@ -106,10 +110,11 @@ def run(self, filename, file,
 
         :param timeout:          Request timeout period.
         :param link_ignore_regex:     A regex for urls to ignore.
+        :param link_ignore_list: Comma separated url globs to ignore
         :param follow_redirects: Set to true to autocorrect redirects.
```
Comment (Asnelchristian) [inline bears/general/InvalidLinkBear.py]: can you open an issue to align this param docs correctly ?

**Comment by Asnelchristian** [comment]
ack https://github.com/coala/coala-bears/pull/1174/commits/0deaeb7ff6b7d8cd6a2c3a1bb7ce91326b7cbff8

**Comment by Asnelchristian** [comment]
ack https://github.com/coala/coala-bears/pull/1174/commits/3011d412190371828233db3b3d69c3ad7d8e67e6

**Comment by Asnelchristian** [comment]
you need a rebase

**Comment by ashwani99** [comment]
@Asnelchristian I have rebased it.

**Comment by Mixih** [comment]
ack 8fe82fd

**Comment by Mixih** [comment]
let's get this done

**Comment by Mixih** [comment]
@rultor merge

**Pull request merged**